pkg/common/cns-lib/volume/manager.go

```diff
@@ -55,6 +55,9 @@ const (
 	// used only for listView
 	noTimeout = 0 * time.Minute
 
+	// DefaultOperationTimeoutInSeconds specifies the default CSI operation timeout in seconds
+	DefaultOperationTimeoutInSeconds = 300
```

Comment:
If the value is not configurable, I think the variable name need not be prefixed with `Default`.
nit: Can be renamed too `VolumeOperationTimeoutInSeconds`

Reply:
Changed to `VolumeOperationTimeoutInSeconds`